src/common/components/PlaceholderFactory.tsx

```diff
@@ -12,7 +12,7 @@ export const PlaceholderFactory = (name: string) => {
   const Placeholder: FC<PlaceholderInterface> = (props) => {
     const invalidEnvironment =
```

Comment:
// This placeholder factory makes components that can be used as stubs for
// development. There should be no Placeholders in production, so an error is
// thrown in that case.
There are 2 production environments, appdev, and production, if that matters